pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -133,7 +133,14 @@ func (a *Actuator) Create(cluster *clusterv1.Cluster, machine *clusterv1.Machine
 		return errors.Wrap(err, "failed to reconcile LB attachment")
 	}
 
-	if err := a.storeMachineStatus(machine, status, true); err != nil {
+	if err := a.updateMachine(machine); err != nil {
+		glog.Errorf("failed to update machine %q in namespace %q trying to set annotation: %v", machine.Name, machine.Namespace, err)
+		return &controllerError.RequeueAfterError{
+			RequeueAfter: time.Minute,
+		}
+	}
+
+	if err := a.storeMachineStatus(machine, status); err != nil {
```

Comment:
The machine update above should also take care of updating the status, is that not the case?

Reply:
That's correct, that is not the case. I suspect it has something to do with CRDs and subresources, but I'm unfamiliar with the depths of CRDs